src/index.js

```diff
@@ -1,9 +1,36 @@
-var core = module.exports = require('./core');
-
-// plugins:
-core.extras         = require('./extras');
-core.filters        = require('./filters');
-core.interaction    = require('./interaction');
-core.loaders        = require('./loaders');
-core.spine          = require('./spine');
-core.text           = require('./text');
+var core = require('./core'),
+	math = core.math,
+	// plugins:
+	extras         = require('./extras'),
+	filters        = require('./filters'),
+	interaction    = require('./interaction'),
+	loaders        = require('./loaders'),
+	spine          = require('./spine'),
+	text           = require('./text'),
+    deprecation    = require('./deprecation');
+
+var modules = [];
+
+modules.push(math);
+modules.push(extras);
+modules.push(filters);
+modules.push(interaction);
+modules.push(loaders);
+modules.push(spine);
+modules.push(text);
+
+for (var i = 0; i < modules.length; i++) {
+
+	// Adds the module to the core, so you can do
+	// PIXI.Text instead of PIXI.text.Text
+	bundleUp(core,modules[i]);
+};
+
+function bundleUp(object1,object2){
+
+    for(var keys in object2){
+        object1[keys] = object2[keys];
+    }
+}
+
+module.exports = core;
```

Comment:
I like having things in their plugin namespace. If you want to alias things to the root object, it should be in the compatibility layer not the default v3 build imho. If the goal is easier compat with v2, it makes sense to be in the compat layer which people can opt **into** (not out of).